tangent/grads.py

```diff
@@ -301,7 +301,15 @@ def maximum(ans, x, y):
 def aarray(ans,x):
   d[x] = tangent.astype(d[ans],x)
 
+  
+@adjoint(numpy.linalg.det)
+def adet(z, x):
+  """d|A|/dA = adj(A).T
 
+  See  Jacobi's formula: https://en.wikipedia.org/wiki/Jacobi%27s_formula
+  """
+  adjugate = np.linalg.det(x) * np.linalg.pinv(x)
+  d[x] = adjugate.T
```

Comment:
This should be np.transpose(adjugate) in order to take higher-order derivatives. Also, if you add np.linalg.pinv(x), grad of grad should "just work"!